<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Случайный массив</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_lnt">
      <property name="text">
       <string>Введите размер массива:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_lnt"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_rng">
      <property name="text">
       <string>Введите диапазон значений:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_rng"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="button_create">
      <property name="text">
       <string>Создать массив</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="button_sum">
      <property name="text">
       <string>Суммировать</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPlainTextEdit" name="list_output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="active_load"/>
   </widget>
   <widget class="QMenu" name="menu_req">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="author"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_req"/>
  </widget>
  <action name="author">
   <property name="text">
    <string>О разработчике...</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить в файл...</string>
   </property>
  </action>
  <action name="active_load">
   <property name="text">
    <string>Загрузить из файла...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
